VMBackup/main/parameterparser.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, protected_settings, public_settings, backup_logger):
         self.includedDisks = None
         self.dynamicConfigsFromCRP = None
         self.wellKnownSettingFlags = {CommonVariables.isSnapshotTtlEnabled: False, CommonVariables.useMccfToFetchDsasForAllDisks: False, CommonVariables.useMccfForLad: False}
+        self.settingKeysMapping= {"issnapshotttlenabled": CommonVariables.isSnapshotTtlEnabled, "usemccftofetchdsasforalldisks": CommonVariables.useMccfToFetchDsasForAllDisks, "usemccfforlad": CommonVariables.useMccfForLad}
```

Comment:
1. instead of hardcoding the names in lowercase here maybe do something like
```python
self.settingKeyMappings = {}
self.settingKeyMappings[CommonVariables.isSnapshotTtlEnabled.lower()] = CommonVariables.isSnapshotTtlEnabled
# ...
# and so on
```
That way the any change to the value associated with the key needs to be done at one place, in case the key itself does not change.

2. Also review if this value needs to be a class level attribute. Maybe use a local variable if this should/need not be accessible outside this class.